### DISCUSSION TITLE
Core Autocorrect Support on iOS

### DISCUSSION THREAD

**Discussion by transcendr** [Flutter-Bounty-Hunters/super_editor#700] [General]
Hello, I'm starting this discussion in regards to the support of autocorrect features on ios.  The behavior I am referring to is as you type, if you enter a typo, it's typical for the word to be automatically corrected to the proper spelling as you move to the next one. Also, the predictive text toolbar should appear above the keyboard if enabled in ios keyboard settings.  

Our current editor package does not support it, and as we are considering switching to SE, upon testing, noticed that typos are not autocorrected in SE either.  Is this on your guys' radar?  

Thanks so much in advance!

**Comment by matthew-carroll** [comment]
We have an existing ticket for autocorrect, here: https://github.com/superlistapp/super_editor/issues/388

It's unclear when we'll get to that work. There may be blocking Flutter issues along the way. I'm not sure.
